app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,35 @@
+// Place all the styles related to the Admin::CafeWorks controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.calendar-tooltip {
+  width: auto;
+  height: auto;
+  background: #eb7125;
+  position: absolute;
+  z-index: 10001;
+  padding: 10px;
+  padding-top: 0px;
+  border: 2px solid;
+}
+
+.cafe-date {
+  width: 100px;
+  height: 85px;
+  color: #fff;
+  float: center;
+  text-align: center;
+  background: #eb7125;
```

Comment:
Color literals like `#eb7125` should only be used in variable declarations; they should be referred to via variable everywhere else.